src/Foundation/Connect/code/Mappers/Profiles/AddressProfile.cs

```diff
@@ -0,0 +1,35 @@
+﻿//    Copyright 2020 EPAM Systems, Inc.
+// 
+//    Licensed under the Apache License, Version 2.0 (the "License");
+//    you may not use this file except in compliance with the License.
+//    You may obtain a copy of the License at
+// 
+//      http://www.apache.org/licenses/LICENSE-2.0
+// 
+//    Unless required by applicable law or agreed to in writing, software
+//    distributed under the License is distributed on an "AS IS" BASIS,
+//    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//    See the License for the specific language governing permissions and
+//    limitations under the License.
+
+namespace Wooli.Foundation.Connect.Mappers.Profiles
+{
+    using AutoMapper;
+
+    using Models;
+
+    using Sitecore.Commerce.Engine.Connect.Entities;
+
+    public class AddressProfile : Profile
+    {
+        public AddressProfile()
+        {
+            this.CreateMap<Party, CommerceParty>()
+                .ForMember(dest => dest.UserProfileAddressId, opt => opt.Ignore())
+                .ForMember(dest => dest.EveningPhoneNumber, opt => opt.Ignore())
+                .ForMember(dest => dest.FaxNumber, opt => opt.Ignore())
+                .ForMember(dest => dest.RegionCode, opt => opt.Ignore())
+                .ForMember(dest => dest.RegionName, opt => opt.Ignore());
```

Comment:
Maybe it's easier to enumerate fields for mapping and then use `.ForAllOtherMembers(opt => opt.Ignore())`?

Reply:
You will have to enumerate more properties to map then to ignore with such approach